squad/core/management/commands/compute_build_summaries.py

```diff
@@ -55,6 +50,11 @@ def add_arguments(self, parser):
             help="End date for project status updates (default: today, format: YYYY-MM-DD)."
         )
 
+    def __progress__(self, show):
+        if show:
+            self.stdout.write(".", ending="")
+            self.stdout._out.flush()
```

Comment:
just fyi, you can do `print('.', end='', flush=True)` for that...

Reply:
It was like this before and it didn't work. I guess it's because django uses additional caching.